### ISSUE TITLE
executor.queueSize=1 doesn't handle multi-cpu processes

### ISSUE THREAD

**Issue by fmorency** [nextflow-io/nextflow#246]
Nextflow is unable to execute processes with `cpus > 1` when `executor.queueSize=1`. Error message is

> ERROR ~ Error executing process > XYZ (id)
> Caused by:
> Process requirement exceed available CPUs -- req: 7; avail: 1

Nextflow should be able to execute multi-cpu processes, one process at a time.

**Comment by pditommaso** [comment]
I've uploaded a snapshot containing a patch for this issue. To use is run 

```
CAPSULE_RESET=1 NXF_VER=0.22.5-SNAPSHOT nextflow info
```

Then use nextflow as shown below: 

```
NXF_VER=0.22.5-SNAPSHOT nextflow run ..
```

**Comment by pditommaso** [comment]
Included in version `0.22.5`
